tests/test_optim.py

```diff
@@ -66,40 +66,40 @@ def test_minimize(tensorlib, precision, optimizer, do_grad, do_stitch):
             'no_grad-scipy-tensorflow-32b': [1.0, 1.0],
             'no_grad-scipy-jax-32b': [1.0, 1.0],
             # no grad, scipy, 64b
-            'no_grad-scipy-numpy-64b': [0.49998815367220306, 0.9999696999038924],
-            'no_grad-scipy-pytorch-64b': [0.49998815367220306, 0.9999696999038924],
-            'no_grad-scipy-tensorflow-64b': [0.49998865164653106, 0.9999696533705097],
-            'no_grad-scipy-jax-64b': [0.4999880886490433, 0.9999696971774877],
+            'no_grad-scipy-numpy-64b': [0.4999985, 1.00000122],
+            'no_grad-scipy-pytorch-64b': [0.4999985, 1.00000122],
+            'no_grad-scipy-tensorflow-64b': [0.49999869, 1.00000121],
+            'no_grad-scipy-jax-64b': [0.49999894, 1.00000114],
             # do grad, scipy, 32b
-            'do_grad-scipy-pytorch-32b': [0.49993881583213806, 1.0001085996627808],
-            'do_grad-scipy-tensorflow-32b': [0.4999384582042694, 1.0001084804534912],
-            'do_grad-scipy-jax-32b': [0.4999389052391052, 1.0001085996627808],
+            'do_grad-scipy-pytorch-32b': [0.49999850, 1.00000131],
+            'do_grad-scipy-tensorflow-32b': [0.49999896, 1.0000011],
+            'do_grad-scipy-jax-32b': [0.5003347, 1.0001591],
             # do grad, scipy, 64b
-            'do_grad-scipy-pytorch-64b': [0.49998837853531425, 0.9999696648069287],
-            'do_grad-scipy-tensorflow-64b': [0.4999883785353142, 0.9999696648069278],
-            'do_grad-scipy-jax-64b': [0.49998837853531414, 0.9999696648069285],
+            'do_grad-scipy-pytorch-64b': [0.499998588, 1.000001218],
+            'do_grad-scipy-tensorflow-64b': [0.49999859, 1.00000122],
+            'do_grad-scipy-jax-64b': [0.49999859, 1.00000122],
             # no grad, minuit, 32b - not very consistent for pytorch
-            'no_grad-minuit-numpy-32b': [0.49622172117233276, 1.0007264614105225],
+            'no_grad-minuit-numpy-32b': [0.5021529, 0.99905187],
             #    nb: macos gives different numerics than CI
             #'no_grad-minuit-pytorch-32b': [0.7465415000915527, 0.8796938061714172],
             'no_grad-minuit-pytorch-32b': [0.9684963226318359, 0.9171305894851685],
-            'no_grad-minuit-tensorflow-32b': [0.5284154415130615, 0.9911751747131348],
+            'no_grad-minuit-tensorflow-32b': [0.5073193, 0.99639267],
```

Comment:
@kratsg @lukasheinrich I expected there to be some changes, but I'm confused and a bit worried that to CI to pass I had to introduce such large changes as ones like this here. We should probably think on this a bit, but if either of you have thoughts on why I'd be very interested to hear them.

Reply:
it's 32bit. It's... ... tricky.

Reply:
> it's... ... tricky.

:slightly_smiling_face: true. So what's your thoughts here?

Reply:
This change seems to currently not be included in the PR anymore, is the change no longer needed? It seemed rather large, especially compared to everything else changing.

Reply:
@alexander-held there is still a large relative tolerance under `if "minuit-tensorflow-32b" in identifier:` in `tests/test_optim.py` in the PR, but this specific changing of the expected hasn't been set again since I rebased it.

Reply:
Resolved by removal in PR #1504